### ISSUE TITLE
replace remove the end string

### ISSUE THREAD

**Issue by miewx** [Nugine/const-str#11]
0.4.2

const_str::replace!(stringify!($file), " ", "");  

"run / v4" =>  "run/"
"token" => ""

<img width="1426" alt="image" src="https://user-images.githubusercontent.com/26247733/176839590-85fcfd9e-0960-4e9d-b64d-de0eec6b6223.png">

rollback to 0.3 everything is ok

**Comment by Nugine** [comment]
Introduced in 102a532ba1c7b506007730763fb240615d61f80e
Fixed in cc367a92475aa7868a87d198dc15ea7a26c7e237 (latest commit)

You can test it locally by adding a github dependency:
```toml
const-str = { git = "https://github.com/Nugine/const-str", rev = "cc367a92475aa7868a87d198dc15ea7a26c7e237" }
```

If it works, I will publish const-str v0.4.3.

**Comment by Nugine** [comment]
const-str v0.4.3 has been released.
